lib/hcloud/client.rb

```diff
@@ -108,6 +108,27 @@ def __getobj__
       end
     end
 
+    def multi
+      @multi = true
+      yield
+    ensure
+      @multi = false
+    end
+
+    def response(request, json: true, &block)
+      raise 'No block given' unless block_given?
+
+      if @multi
+        hydra.queue request
+        sleep 1 while request.response.nil?
+      else
```

Comment:
The `multi` stuff is currently unused and can be dropped for this PR, right @tonobo? I copy🐱 ed it from your [ancient branch](https://github.com/tonobo/hcloud-ruby/commit/75c15279eb599127a1addbf060d76ea843459d52).

Reply:
Not sure, i thought it's being used in certain conditions. Have to look if any user benefits from the feature

Reply:
This will be dropped.

As discussed personally, this isn't needed for this PR and should be already implemented via concurrency.